### PR TITLE
tweak SEO of package detail page

### PR DESCRIPTION
**Pull request by g-plane**
When package information fetched at package detail page, title and description of the page will be updated. Once leave the page, title and description will be restored.

### CODE REVIEW

**Review by t8 — COMMENTED** [review]
This is awesome! I only have a few comments regarding these additions:

- I don't think we need to display the description on the header of the package detail page. This may be a generalization, but most READMEs of modules show this information. Curious to hear other opinions on this.
- From a design standpoint, I would prefer to see the back arrow vertically align with the title:
image

Thank you so much for updating the title & descriptions in SEO of these pages. This is incredibly useful. Have you tested this on Twitter?

**Comment by g-plane** [comment]
No, because I'm not able to test it before the preview deployment.

**Comment by g-plane** [comment]
Oh, it doesn't work. Maybe the reason is that the package info is fetched at runtime, and the soical services can't wait for it.

**Comment by t8** [comment]
Yes, I think that is the issue. @martonlederer has tried to get around this, but if I remember correctly, we came to the conclusion that SSR would be necessary for this capability to exist on the site.

@g-plane, would you mind removing the dynamic Twitter pieces (and whatever else is affected)? I'd still like to merge the pieces that do work (ex: the title changing).

**Comment by martonlederer** [comment]
Yeah, we will probably do this SSR in 1.0

File: src/views/PackageDetail.vue [hunk 1]
```diff
@@ -13,12 +13,13 @@
       </div>
       <div class="hero-body">
         <div class="container">
+          <router-link to="/gallery" class="back-arrow title">
```
Comment (t8) [inline src/views/PackageDetail.vue:16]: Thanks for making these changes! Final thing: I would prefer the header of this page to be how it was originally formatted, without the description.

The reason is that the description field is redundant on when it's used on module READMEs as well.

Example: 
image

**Review by t8 — APPROVED** [review]
LGTM

Thank you for the contribution @g-plane!

**Pull request merged**